test/schemas.js

```diff
@@ -418,7 +418,61 @@ describe('Schemas', function() {
     });
   });
 
-  it('should override the saved value if min and max value has been set and the submitted value is out of range for a integer field type', function(done) {
+  it('should allow saving of a 0 value if there is no min value set for an integer type field', function(done) {
+    var schema = apos.schemas.compose({
+      addFields: [
+        {
+          type: 'integer',
+          name: 'price',
+          label: 'Price'
+        }
+      ]
+    });
+    assert(schema.length === 1);
+    var input = {
+      price: '0'
+    };
+    var req = apos.tasks.getReq();
+    var result = {};
+    // var result = { password: 'serious' };
+    return apos.schemas.convert(req, schema, 'form', input, result, function(err) {
+      assert(!err);
+      assert(_.keys(result).length === 1);
+      // hashing is not the business of schemas, see the
+      // apostrophe-users module
+      assert(result.price === 0);
+      done();
+    });
+  });
+
+  it('should allow saving of a 0 value if there is no min value set for a float type field', function(done) {
+    var schema = apos.schemas.compose({
+      addFields: [
+        {
+          type: 'float',
+          name: 'price',
+          label: 'Price'
+        }
+      ]
+    });
+    assert(schema.length === 1);
+    var input = {
+      price: '0'
+    };
+    var req = apos.tasks.getReq();
+    var result = {};
+    // var result = { password: 'serious' };
+    return apos.schemas.convert(req, schema, 'form', input, result, function(err) {
+      assert(!err);
+      assert(_.keys(result).length === 1);
+      // hashing is not the business of schemas, see the
```

Comment:
Can lose this comment too. Was meant to explain why the password schema field appears to save plaintext (the users module takes care of converting it to a hash before it's actually saved). This isn't a password field so not in play here.

Reply:
Yeah, copy / paste, will review all the tests I added to remove any extra comments.